odoo/addons/base/models/res_users.py

```diff
@@ -1435,6 +1439,8 @@ def _should_alert_new_device(self):
 # to the implied groups (transitively).
 #
 
+
+# pylint: disable=E0102
```

Comment:
Can you add a TODO in the file so that we split this file into multiple smaller files and avoid E0102 altogether?

Reply:
Or merge the user classes ?
(After the pr of group change, I remove one class, maybe the other will be merge)